src/units/zone_server.rs

```diff
@@ -520,21 +502,15 @@ impl ZoneServer {
 
         info!("[{unit_name}]: Seeking approval for {zone_type} zone '{zone_name}' at serial {zone_serial}.");
 
-        // Only if not already approved...
-        let approval_token = Uuid::new_v4();
-        info!("[{unit_name}]: Generated approval token '{approval_token}' for {zone_type} zone '{zone_name}' at serial {zone_serial}.");
-
         self.pending_approvals
             .write()
             .await
-            .entry((zone_name.clone(), zone_serial))
-            .and_modify(|e| e.push(approval_token))
-            .or_insert(vec![approval_token]);
+            .insert((zone_name.clone(), zone_serial));
 
         let Some(hook) = review.cmd_hook else {
             info!("[{unit_name}] No review hook set; waiting for manual review");
-            info!("[{unit_name}]: Confirm with HTTP GET {}approve/{approval_token}?zone={zone_name}&serial={zone_serial}", self.http_api_path);
-            info!("[{unit_name}]: Reject with HTTP GET {}reject/{approval_token}?zone={zone_name}&serial={zone_serial}", self.http_api_path);
+            info!("[{unit_name}]: Approve with HTTP POST /zone/{zone_name}/{zone_type}/{zone_serial}/approve");
```

Comment:
Is this guidance correct? Do we want people approving by using the otherwise undocumented and unofficial, internal, HTTP API? Don't we want to guide people to do `cascade zone approve` or whatever instead?

Reply:
I changed this to show the `cascade zone approve` and `cascade zone reject` commands that an operator would need to use.